src/Angor/Client/Pages/Settings.razor

```diff
@@ -471,39 +472,56 @@
             await Refresh(false);
         }
     }
-
+    
     private async Task AddIndexer()
     {
         if (!string.IsNullOrWhiteSpace(newIndexerLink))
         {
             if (!Uri.TryCreate(newIndexerLink, UriKind.Absolute, out Uri? uri))
             {
-                notificationComponent.ShowErrorMessage($"invalid url {newIndexerLink}");
+                notificationComponent.ShowErrorMessage($"Invalid URL: {newIndexerLink}");
                 return;
             }
 
-            if (uri.Scheme is not ("http" or "https"))
```

Comment:
why remove the schema test? cant remember why we put it there, probably to not confuse with relays